backend/auth/Auth.ts

```diff
@@ -28,12 +34,33 @@ let passportStrategy = new Strategy(async function (email, password, done) {
   }
 });
 
+passport.use(
+  new GoogleStrategy(
+    {
+      clientID: process.env.GOOGLE_CLIENT_ID,
+      clientSecret: process.env.GOOGLE_CLIENT_SECRET,
+      callbackURL:
+        process.env.GOOGLE_CALLBACK_URL ||
+        "http://localhost:7600/auth/google/callback", // How to handle this?
```

Comment:
This is fine. The other way is to add GOOGLE_CALLBACK_URL to the .env file in each person's version of the code